js/src/redux/providers/registry/middleware.js

```diff
@@ -14,13 +14,40 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import { debounce } from 'lodash';
 import Contracts from '~/contracts';
 import subscribeToEvents from '~/util/subscribe-to-events';
 
 import registryABI from '~/contracts/abi/registry.json';
 
 import { setReverse, startCachingReverses } from './actions';
 
+const read = (chain) => {
+  const reverses = window.localStorage.getItem(`${chain}-registry-reverses`);
+  const lastBlock = window.localStorage.getItem(`${chain}-registry-reverses-last-block`);
+  if (!reverses || !lastBlock) {
```

Comment:
OCD-related comment (plus I'm old) - some spacing between vars and action (i.e. blank line here) would be appreciated.